docker-compose.yml

```diff
@@ -1,11 +1,22 @@
-version: '3.7'
+version: "3.7"
 
 services:
+  influxdb:
+    image: influxdb:latest
+    container_name: influxdb
+    ports:
+      - "8086:8086"
+
   tsm:
     image: tsm
-    env_file: 
+    env_file:
       - .env
+    environment:
+      - INFLUX_HOST=influxdb
     build: .
     container_name: tsm
     ports:
-      - ${TSM_PORT}:${TSM_PORT}
\ No newline at end of file
+      - ${TSM_PORT}:${TSM_PORT}
+    links:
```

Comment:
I thought links were already removed :)

afaik links were deprecated and we should use networks.